ANONYMITY_NETWORKS.md

```diff
@@ -222,3 +222,71 @@ leak information to hidden services if done improperly.
 At the current time, if users need to frequently make transactions, I2P/Tor
 will improve privacy from ISPs and other common adversaries, but still have
 some metadata leakages to unknown hidden service operators.
+
+# Example: Configure a remote, linux daemon that you can connect to via tor from a linux or mac laptop
+
+Below we'll show an example configuration that allows you to run a remote daemon (eg on a VPS) that you can connect to from other computers using the cli wallet to retrieve blocks over tor.  In the future, the GUI will also allow this.  Because Tor offers encryption and authentication, you can be confident that your RPC credentials will not be sent in the clear.  This setup will also obfuscate the fact that you are connecting to a remote Monero node. Tested with `v0.15.0.1`, remote Linux node and Mac laptop wallet.
```

Comment:
I would expand on the "eg on a VPS". This configuration allows someone to run a node at home without any port forwarding, static dns or dyndns. Eventually full-wallet mobile wallets like CakeWallet could connect to a home daemon this way with nothing more than the onion address.

Reply:
Yes, makes sense to emphasize that this can (and ideally _should_) be run on hardware you control.

> CakeWallet could connect to a home daemon

Monerujo should also work.

Reply:
I adjusted a bit and mentioned Monerujo (which should work fine today).  I have no experience with CakeWallet, so left that out for now.  Ideally, we should test both and add specific examples (with screen shots even).